libs/pubsub/pubsub.go

```diff
@@ -63,7 +63,7 @@ var (
 
 // Query defines an interface for a query to be used for subscribing.
 type Query interface {
-	Matches(tags map[string]string) bool
+	Matches(events map[string][]string) bool
```

Comment:
Would benefit from documentation as to why this `map[string][]string` differs from what's in the ABCI format, which is essentially `[]map[string]string`. 

The latter case allows event types to repeat themselves, with the same set of keys and different values.

The former case consolidates all values for  a given `type+key` into a single list.

Reply:
Hmmm, how does it differ from the ABCI format? The ABCI format is a list of KV pairs, which is essentially a list of values for each given key in `attributes`.

In either case, I've updated the godoc to reflect this change 👍